files/build_templates/syncd.service.j2

```diff
@@ -13,12 +13,14 @@ After=opennsl-modules.service
 {% elif sonic_asic_platform == 'nephos' %}
 After=nps-modules-4.9.0-8-2-amd64.service
 {% endif %}
+After=swss.service
 Before=ntp-config.service
 
 [Service]
 User=root
 Environment=sonic_asic_platform={{ sonic_asic_platform }}
-ExecStart=/usr/local/bin/syncd.sh start
+ExecStartPre=/usr/local/bin/syncd.sh start
+ExecStart=/usr/local/bin/syncd.sh attach
```

Comment:
use wait

Reply:
Discussed offline that we should open a separate PR to change over to `docker wait`